rasa/server.py

```diff
@@ -1292,7 +1292,9 @@ async def parse(request: Request) -> HTTPResponse:
             data = emulator.normalise_request_json(request.json)
             try:
                 parsed_data = await app.agent.parse_message_using_nlu_interpreter(
-                    data.get("text")
+                    data.get("text"),
+                    None,
+                    data.get("time_zone")
```

Comment:
I'd also use the entire `metadata` dict here, what do you think? 
```suggestion
                    data.get("metadata")
```

Reply:
Note that you also need to adapt the implementation of `normalise_request_json`

Reply:
I'm not very much in favour of this change as this would mean changing the implementation of `normalise_request_json()` to return everything other than 'text', 'model' and 'time' in metadata dictionary. Also from my understanding the purpose of `normalise_request_json()` is to just normalise the json request body so it shouldn't have to deal with creation of any metadata.

I can still make the change if you say so.

Reply:
> this would mean changing the implementation of normalise_request_json() to return everything other than 'text', 'model' and 'time' in metadata dictionary.

On the other hand this would make the `metadata` applicable in general and would avoid that we'd have to do another change to the codebase if some user wants to expose some other piece of metadata to some components of theirs. I'd prefer going with the more generally applicable case.

Reply:
Ok, makes sense. I've done that change now.